code/languages/python/tuple/README.md

```diff
@@ -0,0 +1,42 @@
+# Tuple
+
+Tuple is a collection of Python objects much like a list. The sequence of values stored in a tuple can be of any type, and they are indexed by integers. The important difference between a list and a tuple is that tuples are immutable. 
+
+Tuples are immutable, and usually, they contain a sequence of heterogeneous elements that are accessed via unpacking or indexing (or even by attribute in the case of named tuples). Lists are mutable, and their elements are usually homogeneous and are accessed by iterating over the list.
+
+### Defining and Using Tuples
+
+* Tuples are identical to lists in all respects, except for the following properties:
+
+* Tuples are defined by enclosing the elements in parentheses (()) instead of square brackets ([]).
```

Comment:
Add these points as sub-lists of the above sentence.

Reply:
Done!